app/components/Dialog/messages.js

```diff
@@ -0,0 +1,11 @@
+import { defineMessages } from 'react-intl'
+
+/* eslint-disable max-len */
+export default defineMessages({
+  lnd_crashed_dialog_header: 'Lnd has unexpectedly quit!',
+  lnd_crashed_dialog_code: 'Error code:',
+  lnd_crashed_dialog_signal: 'Error signal:',
+  lnd_crashed_dialog_messgage: 'Error message:',
+  lnd_crashed_dialog_unknown: 'unknown',
```

Comment:
First letter should be capitalized for consistency

Reply:
Updated